src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -182,6 +185,22 @@ export default class TutorialView {
         return this;
     }
 
+    /**
+     * Reloads the tutorial when the airport is changed.
+     *
+     * @for tutorialView
+     * @method onAirportChange
+     */
+    onAirportChange() {
+        const step = prop.tutorial.step;
```

Comment:
what is `step` here?  is this the `currentStep`?  meaning the current position in the tutorial?

Reply:
I decided to remove that code as it causes new issues. If you are switching to another airport, the tutorial should now reset to the first page.